package.json

```diff
@@ -24,26 +24,26 @@
   "author": "Vojta Jina <vojta.jina@gmail.com>",
   "dependencies": {},
   "devDependencies": {
-    "chai": "^4.1.2",
-    "eslint": "^4.18.2",
-    "eslint-config-standard": "^11.0.0",
-    "eslint-plugin-import": "^2.9.0",
-    "eslint-plugin-node": "^6.0.1",
-    "eslint-plugin-promise": "^3.7.0",
-    "eslint-plugin-standard": "^3.0.1",
-    "grunt": "^1.0.1",
+    "chai": "^4.2.0",
+    "eslint": "^5.16.0",
+    "eslint-config-standard": "^12.0.0",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-node": "^8.0.1",
+    "eslint-plugin-promise": "^4.0.1",
+    "eslint-plugin-standard": "^4.0.0",
+    "grunt": "^1.0.4",
     "grunt-bump": "^0.8.0",
-    "grunt-eslint": "^20.1.0",
-    "grunt-karma": "2.x",
+    "grunt-eslint": "^21.0.0",
+    "grunt-karma": "3.x",
     "grunt-npm": "0.0.2",
-    "jasmine-core": "^3.1.0",
-    "karma": "^2.0.0",
+    "jasmine-core": "^3.3.0",
+    "karma": "^3.1.4",
     "karma-chrome-launcher": "^2.2.0",
     "karma-firefox-launcher": "1.x || ^0.1.6",
-    "karma-jasmine": "1.x || ^0.3.6",
-    "load-grunt-tasks": "^3.2.0",
-    "mocha": "^5.0.4",
-    "qunit": "^2.5.1"
+    "karma-jasmine": "2.x",
+    "load-grunt-tasks": "^4.0.0",
+    "mocha": "^6.0.2",
+    "qunit": "^2.9.2"
   },
   "peerDependencies": {
     "qunit": "^2.0.0"
```

Comment:
Hmm, shouldn't we include `karma` in peerDeependencies?

Reply:
seems a good idea to me 👍